StyleCop.Analyzers/StyleCop.Analyzers/Helpers/XmlSyntaxFactory.cs

```diff
@@ -191,6 +196,16 @@ public static XmlElementSyntax ReturnsElement(string newLineText, SyntaxList<Xml
             return MultiLineElement(XmlCommentHelper.ReturnsXmlTag, newLineText, content);
         }
 
+        public static XmlElementSyntax CompactReturnsElement(params XmlNodeSyntax[] content)
```

Comment:
:bulb: The previous `ReturnsElement` method is only used by test code. You can replace the current behavior (`MultiLineElement`) with the behavior you added here, and the new methods can be removed.